### PR TITLE
Add VTAdmin Operator's Guide

### PR DESCRIPTION
**Pull request by doeg**
Signed-off-by: Sara Bee <855595+doeg@users.noreply.github.com>

Deploy previews:
- [Operator's Guide](https://deploy-preview-1012--vitess.netlify.app/docs/14.0/reference/vtadmin/operators_guide/)
- [vtadmin-web reference](https://deploy-preview-1012--vitess.netlify.app/docs/14.0/reference/programs/vtadmin-web/)

This does not cover the changes proposed in https://github.com/vitessio/vitess/pull/10543, nor [single-component VTAdmin](https://github.com/vitessio/vitess/pull/10214) (which would simplify this a lot). Neither does it go into much detail about deployment specifics (Docker, k8s, etc.).

### CODE REVIEW

File: content/en/docs/14.0/reference/programs/vtadmin-api.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+| `lmux-read-timeout` | Optional | seconds | 1s | How long to spend connection muxing. | 
+
+### Cluster config flags
+
+One of `cluster`, `cluster-defaults`, or `cluster-config` file is required. Multiple configurations are permitted; precedence is noted below.
+
+| Name | Definition |
+| -------- | --------- | 
+| `cluster` | Per-cluster configuration. Any values here take precedence over those defined by the `cluster-defaults` and/or `cluster-config` flags. | 
+| `cluster-config` | Path to a yaml cluster configuration; see [clusters.example.yaml](https://github.com/vitessio/vitess/blob/main/doc/vtadmin/clusters.yaml). |
```
Comment (ajm188) [inline content/en/docs/14.0/reference/programs/vtadmin-api.md]: one minor thing, is the file you're linking to here is called `clusters.yaml`, not `clusters.example.yaml`. Maybe just say `"see the example [clusters.yaml]"`?

Comment (ajm188) [reply content/en/docs/14.0/reference/programs/vtadmin-api.md]: (i'm pretty sure you copied this straight from the help text i wrote, which is also wrong, and for that i am sorry)

**Review by ajm188 — COMMENTED** [review]
this is looking good!! i like the structure you're setting up with the operators guide page

File: content/en/docs/14.0/reference/programs/vtadmin-api.md (hunk 1 above)
Comment (doeg) [reply content/en/docs/14.0/reference/programs/vtadmin-api.md]: Oh ha thank you for catching that!

**Comment by doeg** [comment]
I'm going to pull out the vtadmin(-api) and vtadmin-web reference pages into separate PRs. The skeletal operator's guide in this branch be changing a bunch once single-component vtadmin lands (https://github.com/vitessio/vitess/issues/10118), but the program reference makes sense to commit now.

File: content/en/docs/14.0/reference/vtadmin/operators_guide.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
+---
+title: Operator's Guide
+description: How to configure and run VTAdmin
+---
+
+{{< info >}}
+If you run into issues or have questions, we recommend posting in our [#feat-vtadmin Slack channel](https://vitess.slack.com). Click the Slack icon in the top right to join. This is a very active community forum and a great place to interact with other users.
```
Comment (ajm188) [inline content/en/docs/14.0/reference/vtadmin/operators_guide.md]: do you have an opinion on linking to the slack URL vs the slack channel URL? (i don't think i do, this just occurred to me as i read this)

File: content/en/docs/14.0/reference/vtadmin/operators_guide.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+This guide describes how to configure and build the VTAdmin API server (`vtadmin`) and front-end (`vtadmin-web`).
+
+The simplest VTAdmin deployment involves a single Vitess cluster. You can look
+at the [local example][local_example] for a
+minimal invocation of the `vtadmin-api` and `vtadmin-web` binaries.
+
+## Prerequisites
+
+- Building `vtadmin-web` requires [node](https://nodejs.org/en/) >= v16.13.0.
```
Comment (ajm188) [inline content/en/docs/14.0/reference/vtadmin/operators_guide.md]: there's _technically_ an issue if you go to `>= 17.0.0` (see https://github.com/vitessio/vitess/blob/main/web/vtadmin/README.md#prerequisites), which we might want to call out here

File: content/en/docs/14.0/reference/vtadmin/operators_guide.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+
+## 3. Configure and build `vtadmin-web`
+
+{{< info >}}
+Unlike other Vitess components, `vtadmin-web` is not distributed as a binary. This is because environment variables specific to a Vitess environment (such as the hostname for `vtadmin-api`) are embedded during build time. Since `vtadmin-web` produces a static HTML/CSS/JS build, it's not possible to read these values at run-time.
+{{< /info >}}
+
+Environment variables can be defined in a `.env` file or passed inline to the `npm run build` command. The full list of flags is given in the [vtadmin-web reference documentation][vtadmin_web_env_ref].
+
+The following is from the [local example][local_example] showing a minimal set of environment variables. `$web_dir`, in this case, refers to the [vtadmin-web source directory][vtadmin_web_src] but could equally apply to the `web/vtadmin/` directory copied into a Docker container, for example. `REACT_APP_VTADMIN_API_ADDRESS` uses
```
Comment (ajm188) [inline content/en/docs/14.0/reference/vtadmin/operators_guide.md]: I think there's something missing here. This section ends with:

> `REACT_APP_VTADMIN_ADDRESS` uses \<end of paragraph\>

**Review by ajm188 — APPROVED** [review]
this looks great to me! (and thank you for putting this together! ❤️), i found one issue i think needs to be addressed, and one other "just curious" question, but i'm happy to merge after that first thing is addressed

File: content/en/docs/14.0/reference/vtadmin/operators_guide.md (hunk 3 above)
Comment (doeg) [reply content/en/docs/14.0/reference/vtadmin/operators_guide.md]: As a side note, it's possible to specify a more precise node version in the [package.json `engine` property](https://github.com/vitessio/vitess/blob/main/web/vtadmin/package.json#L6).

For example:
- `"node": "^16.13.0"` allows versions that does not increment the first non-zero portion of semver; e.g., it includes 16.13, 16.420, and 16.99; it does not include 16.12 or 17.x. (Probably best?)
- `"node": "16.13.0"` disallows any other version.

The version is enforced (albeit weakly + not exactly how you'd expect or want) by way of the [.npmrc file](https://github.com/vitessio/vitess/blob/main/web/vtadmin/.npmrc) and will fail `npm install` when run with an unspecified version. More on that here: https://docs.npmjs.com/cli/v8/using-npm/config#engine-strict

**Review by ajm188 — APPROVED** [review]
looks fantastic! is there anything else you want to get in this PR, or should I (or you!) merge now?

**Comment by doeg** [comment]
@ajm188 I can merge. Thanks for taking another look!

**Pull request merged**